RAGchain/benchmark/dataset/msmarco.py

```diff
@@ -0,0 +1,131 @@
+from copy import deepcopy
+from typing import List, Optional
+
+import pandas as pd
+from datasets import load_dataset
+
+from RAGchain.DB.base import BaseDB
+from RAGchain.benchmark.dataset.base import BaseDatasetEvaluator
+from RAGchain.pipeline.base import BasePipeline
+from RAGchain.retrieval.base import BaseRetrieval
+from RAGchain.schema import EvaluateResult, Passage
+
+
+class MsMarcoEvaluator(BaseDatasetEvaluator):
+    """
+    StrategyQAEvaluator is a class for evaluating pipeline performance on StrategyQA dataset.
+    """
+
+    file_path = "ms_marco"
+    dataset = load_dataset(file_path, 'v1.1')
```

Comment:
And why do you download dataset at here, not __init__? Is there any reason for that? And why do you make dataset to class variable? Maybe someone wants to evaluate two separate evaluate_size on MSMARCO.